frontend/server/src/Controllers/Problem.php

```diff
@@ -5858,12 +5865,96 @@ public static function getLibinteractiveGenForSmarty(\OmegaUp\Request $r): array
      * @return array{smartyProperties: array{payload: CollectionDetailsPayload, title: \OmegaUp\TranslationString}, entrypoint: string}
      *
      * @omegaup-request-param string $collection_type
+     *
+     * @omegaup-request-param null|string $difficulty_range
+     * @omegaup-request-param mixed $language
+     * @omegaup-request-param int|null $max_difficulty
+     * @omegaup-request-param int|null $min_difficulty
+     * @omegaup-request-param int|null $min_visibility
+     * @omegaup-request-param mixed $offset
+     * @omegaup-request-param mixed $only_karel
+     * @omegaup-request-param mixed $order_by
+     * @omegaup-request-param mixed $page
+     * @omegaup-request-param null|string $programming_languages
+     * @omegaup-request-param null|string $query
+     * @omegaup-request-param mixed $require_all_tags
+     * @omegaup-request-param mixed $rowcount
+     * @omegaup-request-param mixed $some_tags
+     * @omegaup-request-param mixed $sort_order
      */
     public static function getCollectionsDetailsForSmarty(\OmegaUp\Request $r): array {
         $collectionType = $r->ensureString('collection_type');
 
         $collection = [];
+        $problems = [];
 
+        // Problem list
+        $offset = null;
+        $pageSize = \OmegaUp\Controllers\Problem::PAGE_SIZE;
+
+        if (is_null($r['page'])) {
+            $offset = is_null($r['offset']) ? 0 : intval($r['offset']);
+        }
+        if (!is_null($r['rowcount'])) {
+            $pageSize = intval($r['rowcount']);
+        }
```

Comment:
estos también deberían usar los `$r->ensureOptionalInt(...)` que mencionó @pabo99 para validar que existen Y asignarles el tipo correcto al mismo tiempo:

```suggestion
        $offset = $r->ensureOptionalInt('offset') ?? 0;
        $pageSize = $r->ensureOptionalInt('rowcount') ?? \OmegaUp\Controllers\Problem::PAGE_SIZE;
```